src/lib/blockchain/BumpkinItems.ts

```diff
@@ -65,3 +64,38 @@ export async function balanceOf(
     throw error;
   }
 }
+
+export async function loadBalanceBatch(
```

Comment:
Since we now have global functions it may be useful naming this specific to Bumpkin Items.

E.g. `loadWearablesBalanceBatch`